Integrations/integration-jira.yml

```diff
@@ -2,7 +2,7 @@ commonfields:
   id: jira
   version: -1
 name: jira
-display: Atlassian Jira
```

Comment:
this is correct, but you should mention in every command description that it is deprecated.